lib/hanami/router.rb

```diff
@@ -27,7 +27,6 @@ class Router
 
     # Routes for inspection
     #
-    # @api private
```

Comment:
@waiting-for-dev Why?

Reply:
If you want to use a formatter different than the default one, you need to do something like:

```ruby
# `router` is an instance of `Hanami::Router`
Hanami::Router::Inspector.new(router: router.routes, formatter: my_formatter).call
```

Does it make sense?